src/Shared/ChangeWaves.cs

```diff
@@ -0,0 +1,93 @@
+using Microsoft.Build.Shared;
+using System;
+
+namespace Microsoft.Build.Utilities
+{
+    /// <summary>
+    /// All waves are enabled by default, meaning all features behind change waves are enabled.
+    /// </summary>
+    public class ChangeWaves
+    {
+        public static readonly string[] AllWaves = { Wave16_8, Wave16_10, Wave17_0 };
+        public const string Wave16_8 = "16.8";
+        public const string Wave16_10 = "16.10";
+        public const string Wave17_0 = "17.0";
+
+        /// <summary>
+        /// Special value indicating that all features behind change-waves should be enabled.
+        /// </summary>
+        public const string EnableAllFeatures = "999.999";
+
+        internal static readonly Version LowestWaveVersion = new Version(AllWaves[0]);
+        internal static readonly Version HighestWaveVersion = new Version(AllWaves[AllWaves.Length - 1]);
+        internal static readonly Version EnableAllFeaturesVersion = new Version(EnableAllFeatures);
```

Comment:
Do we need this if it's the default?

Reply:
It's just a nice-to-have for easy comparison in my out-of-rotation-check function